src/lib/auth.ts

```diff
@@ -1,16 +1,16 @@
-import NextAuth from 'next-auth'
+import NextAuth from 'next-auth';
 import type { NextAuthOptions } from 'next-auth';
 import GoogleProvider from 'next-auth/providers/google';
 
 export const authOptions: NextAuthOptions = {
   providers: [
     GoogleProvider({
-      clientId: process.env.GOOGLE_CLIENT_ID ||'',
-      clientSecret: process.env.GOOGLE_CLIENT_SECRET ||''
-    })
-  ]
-}
+      clientId: process.env?.GOOGLE_CLIENT_ID ?? '',
+      clientSecret: process.env?.GOOGLE_CLIENT_SECRET ?? '',
+    }),
+  ],
+};
 
-const handler = NextAuth(authOptions)
+const handler = NextAuth(authOptions);
 
-export { handler as GET, handler as POST }
\ No newline at end of file
+export { handler as GET, handler as POST };
```

Comment:
This sequence can be deleted entirely. The handler is exported in src/app/api/auth/[...nextauth]/route.ts (which is the correct place)